src/Controller/api/robot_private.h

```diff
@@ -43,6 +43,7 @@ int robot_get_number_of_devices();
 WbDeviceTag robot_get_device_tag(WbDevice *);
 WbDevice *robot_get_robot_device();
 double wb_robot_get_time();
+bool wb_robot_get_synchronization();
```

Comment:
I believe we should not add public API functions into private include files like this `robot_private.h` file, as it is redundant with the public include files. Instead, we should `#include <webots/robot.h>` from the C file. The same probably applies to `wb_robot_get_time()` and maybe other functions.